uint32 x_center
uint32 y_center
uint32 cm_width
uint32 cm_height
uint32 rotation_degrees